### DISCUSSION TITLE
Site-specific parameter calibration

### DISCUSSION THREAD

**Discussion by MiguelMarante** [NatLabRockies/floris#757] [Q&A]
Hello everyone,

I read some discussions in this section about parameter tuning in FLORIS. It is my understanding that most of these model parameters were calibrated using SOWFA simulations, field experiments, etc... It also seems that default parameters give good estimations for power predictions for most scenarios. However, I was just wondering if there is any studies regarding how specific-site calibration could lead to significant differences in AEP estimations. Are there any parameters which are known to be site-specific and others that are site-independent? Or are there any parameters that are usually more important to take into account than others?

Best regards,
Miguel Pereira

**Comment by misi9170** [comment]
Hi @MiguelMarante , thanks for the question! Site-specific tuning is something that we are actively looking into with the new Empirical Gaussian (EmG) model and encouraging people with SCADA data to try it. Automated tuning of the key wake parameters in the EmG model is shown in our companion repository [FLASC](https://github.com/NREL/flasc), see in particular [this example](https://github.com/NREL/flasc/blob/main/examples_smarteole/07_emgauss_scada_tuning_optimization_method.ipynb). 

There is also currently a [pull request open in FLASC](https://github.com/NREL/flasc/pull/127) that will likely be merged into the develop branch in the coming days that splits data by daytime/nighttime and fits the model parameters to the split data set. 

In terms of already-published works on site-specific model tuning, I'm not aware of many (but others may chime in here). A good place to start though would be [Doekemeijer et al.'s 2022 paper in Energies](https://www.mdpi.com/1996-1073/15/6/1964).

**Reply by MiguelMarante** [reply]
Hello @misi9170,

Thank you for responding so quickly.

I will definitely take a look into that new Empirical Gaussian model, but I was wondering about the GCH and CC models. There are model parameters related to wake deflection and wake deficit and I was just wondering if these are left as user inputs because they might be site-specific and its calibration would be essential in order to get good estimates of AEP gains with WFC. 

Best regards,
Miguel

**Reply by misi9170** [reply]
Hi Miguel,

I'd say that for all models (EmG, GCH, CC, etc), the parameters could be refined to better match site-specific conditions. You're right, this is partly why the parameters are directly editable in the input yaml files. Refining the parameters will likely produce a better match to observed AEP, wake losses, and wake deflection than using the defaults. 

That being said, the default parameters for GCH and CC were selected to be our best guesses for general parameters at the time they were implemented, and as you pointed out, they give good estimates for power production in many situations (and particularly the situations they were designed for). So, if no SCADA data is available for comparison, we treat the default parameters as a best estimate. Here again I'll just highlight Doekemeijer et al., which compares GCH (using the default parameters without retuning) to various SCADA records, and shows reasonable (but not perfect) agreement. 

Cheers,
Misha
